### DISCUSSION TITLE
Reduce apps from application list.

### DISCUSSION THREAD

**Discussion by kawsaramin101** [linuxmint/discussions#407] [Ideas]
The Applications list feels very crowded in Linux mint cinnamon. Here are my few suggestions -
1. Software sources, Update manager and Software manager should be one app.
2. Remove all preferences app because they already exist in System settings app. Add a search feature to main menu that still finds the settings from there.
3. File renamer should be Integrated to the file manager.

**Comment by billyswong** [comment]
File Renamer as a separated app allows users in MATE or Xfce to also enjoy it.

I don't use the System Settings App but fire those individual preferences app form start menu directly, usually via typing the name in the start menu search box. I disagree with your 2nd suggestion as this is against my workflow.
